home/views.py

```diff
@@ -9,10 +9,10 @@ def home_create(request):
         form = CreateHomeForm(request.POST)
         if form.is_valid():
             form.save()
```

Comment:
You should handle validation exception. The following error appear when updating dates:
![image](https://user-images.githubusercontent.com/80269612/227581347-50fbf0fc-df0c-41b2-9e8f-122bea5159fb.png)

A form error should appear instead of a exception page

Reply:
Perfect! Now is working fine!